src/Library/demos/Text Input/main.blp

```diff
@@ -0,0 +1,309 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.Clamp {
+  Box {
+    orientation: vertical;
+
+    Label {
+      label: "Text Input";
+      margin-top: 10;
+      margin-bottom: 10;
+
+      styles [
+        "title-1",
+      ]
+    }
+
+    Label {
+      label: "Widgets used for reading text input";
+      margin-bottom: 10;
+    }
+
+    Label {
+      label: "Entry";
+      margin-top: 10;
+      halign: start;
+
+      styles [
+        "heading",
+      ]
+    }
+
+    Label {
+      label: "A simple but versatile widget that allows single line text entry and editing.";
+      margin-top: 10;
+      margin-bottom: 10;
+      halign: start;
+
+      styles [
+        "dim-label",
+      ]
+    }
+
+    FlowBox {
```

Comment:
Plese set the selecton mode to "none" on all the `FlowBox` to avoid this:

![Screenshot from 2023-03-07 19-19-56](https://user-images.githubusercontent.com/19673/223514351-3ef6095a-e86a-417a-9233-e8db80cbdd9c.png)

Reply:
👍🏾